<launch>
  <node name="position_control" pkg="autonomous_offboard" type="position_control.py" output="screen" />
  <node name="action_controller" pkg="autonomous_offboard" type="action_controller.py" output="screen" />
  <node name="goto_position_server" pkg="autonomous_offboard" type="goto_position_server.py" output="screen" />
  <node name="detect_object_server" pkg="autonomous_offboard" type="detect_object_server.py" output="screen" />
  <node name="tensorflow_detection" pkg="autonomous_offboard" type="tensorflow_hw_mvBlueFox_detection.py" output="screen" />
  <node name="short_grippers_server" pkg="autonomous_offboard" type="short_grippers_server.py" output="screen" />
  <node name="long_grippers_server" pkg="autonomous_offboard" type="long_grippers_server.py" output="screen" />
  <node name="coverage_server" pkg="autonomous_offboard" type="coverage_server.py" output="screen" />
  <node name="descend_on_object_server" pkg="autonomous_offboard" type="descend_on_object_server.py" output="screen" />
  <node name="obstacle_detection_control" pkg="autonomous_offobard" type="obstacle_detection_control.py" output="screen" />
</launch>
